symphony-bdk-core/src/main/java/com/symphony/bdk/core/config/model/BdkDatafeedConfig.java

```diff
@@ -0,0 +1,25 @@
+package com.symphony.bdk.core.config.model;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.io.File;
+
+@Getter
+@Setter
+public class BdkDatafeedConfig {
+
+    private String version = "v1";
```

Comment:
why this attribute with this value by default?

Reply:
because this class is used to parse config file by` JSONMapper` and this mapper parse version attribute from json or yaml into String so this attribute should be kept as String